packages/application-shell/src/components/locale-switcher/locale-switcher.tsx

```diff
@@ -14,6 +17,8 @@ type Props = {
   availableLocales: string[];
 };
 
+const LOCALE_SWITCHER_LABEL_ID = 'locale-switcher-label';
```

Comment:
We have a constant field since there is only one instance of this component in the page.